baselines/flwr_baselines/publications/leaf/femnist/femnist_model_test.py

```diff
@@ -0,0 +1,30 @@
+"""FEMNIST model test."""
```

Comment:
This needs to respect the `<file_to_test>_test.py` convention, so it would be `model_test.py`.

Reply:
Done